### DISCUSSION TITLE
Иконка по умолчанию для пунктов меню в сайдбаре

### DISCUSSION THREAD

**Discussion by shevl-dev** [moonshine-software/moonshine#1375] [Ideas]
Возможно идея бредовая, но мне кажется, что пункты без иконок при свёрнутом сайдбаре выглядят совсем не "юзабельно", но при этом если пунктов много, то подбирать иконки под каждый пункт может быть утомительным.

Может есть смысл в конфиге задавать иконку по умолчанию для всех пунктов меню, если не указана конкретная иконка? Можно сделать разные иконки для групп и для элементов.

![image](https://github.com/user-attachments/assets/a45a46c4-91b5-46ca-95f6-f39f7690cfeb)

**Comment by lee-to** [comment]
Надо подумать
